src/rust/e2e-tests/tests/sysmon_log_e2e_test.rs

```diff
@@ -53,6 +65,18 @@ async fn test_sysmon_log_e2e(ctx: &mut E2eTestContext) {
         .await
         .expect("failed to setup the sysmon-generator");
 
+    let mut uid_allocator_client = build_grpc_client(UidAllocatorClientConfig::parse()).await?;
+    uid_allocator_client
+        .create_tenant_keyspace(CreateTenantKeyspaceRequest { tenant_id })
+        .await?;
+
+    let provisioner_client_config = ScyllaProvisionerClientConfig::parse();
+    let mut provisioner_client = build_grpc_client(provisioner_client_config).await?;
+
+    provisioner_client
+        .provision_graph_for_tenant(ProvisionGraphForTenantRequest { tenant_id })
+        .await?;
+
```

Comment:
These should likely be brought into an E2eTestContext function; this will be a very common functionality.